pawn/Interface/Economy.pwn

```diff
@@ -124,10 +124,10 @@ GetEconomyValue(EconomyValueType: type, inputValue = 0) {
         case TeleportWithoutVehicle:            return 1000;
         case TuneCommand:                       return 1250;
         case VehicleCrusherReward:              return 2000;
-        case VehicleExportReward:               return floatround(inputValue /* vehicle health [371-1000] */ * 15 * 1.216);
-        case VehicleExportRewardOwnerShare:     return floatround(inputValue /* vehicle health [371-1000] */ * 15 * 1.216 * 0.1);
-        case WantedLevelAward:                  return 1000 * inputValue /* wanted level stars */;
-        case WantedLevelOwnerShare:             return 10 * inputValue /* wanted level stars */;
+        case VehicleExportReward:               return floatround(inputValue /* vehicle health [371-1000] */ * 60 * 1.45);
```

Comment:
We're increasing this by well over five times. Can we reduce that a little bit, and change the `60` to `45`?

Reply:
Ah Well I have different thoughts. We are reducing this max earning from 87000 to 65250. Are you definitely sure we should reduce it a bit to 45?

Reply:
@RussellLVP, Done it !! Reduced it to 45.